BadHabbit/BadHabbit/UserStories/SingUpScreen/SignUpViewController.swift

```diff
@@ -0,0 +1,170 @@
+//
+//  SignUpViewController.swift
+//  BadHabbit
+//
+//  Created by Viktoria Misiulia on 19/07/2023.
+//
+
+import UIKit
+import SnapKit
+
+protocol SignUpViewProtocol: AnyObject {
+    var startButton: UIButton { get }
```

Comment:
why it's declared in protocol?

Reply:
Because I assumed that I shouldn't declare an empty protocol or wouldn't use the startButtonTapped() function inside as an internal function. Is it correct to add the startButtonTapped() function instead?

Reply:
There is definitely no need to define variables which aren't used. So it's fine to declare empty protocol

Reply:
Variable was removed